app.js

```diff
@@ -2,19 +2,36 @@ const express = require("express");
 const app = express();
 const { getApi } = require("./controllers/api-controllers");
 const { getTopics } = require("./controllers/topic-controllers");
-
+const { getArticle } = require("./controllers/article-controllers");
 app.use(express.json());
 app.get("/api/", getApi);
 app.get("/api/topics", getTopics);
+app.get("/api/articles/:article_id", getArticle);
 
 app.use((request, response, next) => {
-  response.status(404).send({ msg: "page not found" });
+  response.status(404).send({ msg: "Not found" });
   next();
 });
 
+app.use((err, request, response, next) => {
+  if (err.status === 404) {
```

Comment:
We could maybe check if status and msg exist on an err here just to avoid returning undefined or similar if either property is missing. It's entirely possible an error could have a status (500, 400, etc) but no msg if it's not a custom error from us. Just a recommendation for a possible refactor.